scriptfiles-folder/SSS/languages/English

```diff
@@ -64,3 +64,6 @@ REFUELLING=Refuelling...
 TENTREMOVE=Removing tent...
 NEEDWORKBE=You need a workbench to combine those
 NEARBYKEYP=This door is electronically locked controlled by a nearby keypad.
+HOLSTERHOLS=Holstered smth
+HOLSTEREQUI=Equipped Holster
+BAGITEMADDED=Item added to bag
```

Comment:
The holster keys should already be in the latest copy at line 75. The bag line also is but it has one less character at line 71